pkg/mapper/dynamicfile/mapper.go

```diff
@@ -18,6 +18,10 @@ func NewDynamicFileMapper(cfg config.Config) (*DynamicFileMapper, error) {
 	if err != nil {
 		return nil, err
 	}
+	if value, exists := cfg.ReservedPrefixConfig[strings.ToLower(mapper.ModeDynamicFile)]; exists {
```

Comment:
Can you document that the mapper name must be lowercased? Or do you lowercase it before storage?

Reply:
I have removed the lowercase. It is not needed actually. in config file, 'DynamicFile` is passed in https://github.com/kubernetes-sigs/aws-iam-authenticator/pull/568/files#diff-093cb716b58164ce7c35a6cc848450d2eed7a799a55d3a2e863c0d7cd14a1b36R11